src/main/java/io/vertx/core/eventbus/EventBus.java

```diff
@@ -249,5 +248,7 @@
   @GenIgnore
   EventBus unregisterDefaultCodec(Class clazz);
 
+  void start(Handler<AsyncResult<Void>> completionHandler);
```

Comment:
I would move that on an `EventBusInternal` interface as this should only be used by VertxImpl to start the event bus.